script/collect-files.py

```diff
@@ -0,0 +1,26 @@
+import json
+import os
+from pathlib import Path
+
+def load_json_from_path(file_path: Path):
+    with open(file_path) as json_file:
+        return json.load(json_file)
+
+
+def collect_json_files(adapter_dir: Path, output_file_path: str):
+    adapter_files = sorted(adapter_dir.glob("*.json"))
+
+    result = []
+    for file in adapter_files:
+        if "por.json" not in file.name and "por-fast.json" not in file.name:
+            result.append(load_json_from_path(file))
+
+    with open(output_file_path, "w") as f:
+        json.dump({"result": result}, f, indent=4)
```

Comment:
Minor. Just making the variable names more clear.

```suggestion
def collect_json_files(directory: Path, output_file_path: str):
    files = sorted(directory.glob("*.json"))

    result = []
    for file in files:
        if "por.json" not in file.name and "por-fast.json" not in file.name:
            result.append(load_json_from_path(file))

    with open(output_file_path, "w") as f:
        json.dump({"result": result}, f, indent=4)
```